smartsim/_core/entrypoints/service.py

```diff
@@ -42,19 +42,21 @@ class Service(ABC):
     def __init__(
         self,
         as_service: bool = False,
-        cooldown: int = 0,
-        loop_delay: int = 0,
+        cooldown: float = 0,
+        loop_delay: float = 0,
         health_check_frequency: float = 0,
     ) -> None:
         """Initialize the ServiceHost
 
-        :param as_service: Determines if the host will run until shutdown criteria
-        are met or as a run-once instance
-        :param cooldown: Period of time to allow service to run before automatic
-        shutdown, in seconds. A non-zero, positive integer.
-        :param loop_delay: Delay between iterations of the event loop (in seconds)
-        :param health_check_frequency: Delay between calls to a
-        health check handler (in seconds)
+        :param as_service: Determines if the host runs continuously until
```

Comment:
Potentially bad clarifying Q from me... host also means service here? for `as_service: Determines if the host runs`

Reply:
yes. i replaced 'host' with 'service' and updated all param/member descriptions to be consistent.